src/translations/pt.json

```
@@ -0,0 +1,41 @@
+{
+  "log_in": "Entrar",
+  "log_out": "Sair",
+  "logged_in_as": "Logado como",
+  "logged_in": "Logado em",
+  "logging_in": "Logando em",
+  "logging_out": "Saindo",
+  "sign_up": "Registrar",
+  "signing_up": "Registrando",
+  "forgot_password": "Esqueceu a senha?",
+  "recover_password": "Recuperar senha",
+  "send_recovery_email": "Enviar email de recuperação de senha",
+  "sending_recovery_email": "Enviando email de recuperação de senha",
+  "never_mind": "Deixa pra lá",
+  "update_password": "Atualizar senha",
+  "updating_password": "Atualizando senha",
+  "complete_your_signup": "Complete seu registro",
+  "site_url_title": "Configurações de desenvolvimento",
+  "site_url_link_text": "Limpar URL local",
```

```suggestion
  "site_url_link_text": "Limpar URL do localhost",
```
Thanks! I'm wondering whether this line should refer to the `localhost` explicitly?

---

Thank you. Your suggestion works better.